SBR.md

```diff
@@ -2110,19 +2110,19 @@ i. __Certificate Field:__ `subject:title` (2.5.4.12)
    __Contents:__ If present, the `subject:title` field SHALL contain only a organizational role/title or a regulated professional designation verified according to [Section 3.2.4](#324-authentication-of-individual-identity).
 
 j. __Certificate Field:__ Number and street: `subject:streetAddress` (OID: 2.5.4.9)  
- __Contents:__ If present, the `subject:streetAddress` field SHALL contain the Subject's street address information as verified under [Section 3.2.3](#323-authentication-of-organization-identity) foror [Section 3.2.4](#324-authentication-of-individual-identity).
+ __Contents:__ If present, the `subject:streetAddress` field SHALL contain the Subject's street address information as verified under [Section 3.2.3](#323-authentication-of-organization-identity) for Organization-validated and Sponsor-validated Certificate Types or [Section 3.2.4](#324-authentication-of-individual-identity) for Individual-validated Certificate Types.
```

Comment:
I think this description clarifies the expectations. One small observation is that there is no definition of "Certificate Type" under section 1.6.1 but is described in 1.2. Perhaps, for consistency, we should also add it in 1.6.1 which is the place to look for all capitalized terms throughout the document.